whiz.yaml

```diff
@@ -118,7 +118,7 @@ website_docgen:
         npx docusaurus docs:generate:graphql:prisma-migration
         npx docusaurus docs:generate:changelog
     depends_on:
-        - typegate1
```

Comment:
Does not this depend on `typegate`?

Reply:
Typegate never finishes. I always thought that Whiz should have a way to manage those kind of task : e.g. if no stdout is detected after a while and the task is marked a "continuous".

Reply:
You are right. Something we could implement is to check when there is a server listening on some port.